api/database/seeders/UserSeederUat.php

```diff
@@ -16,32 +16,187 @@ class UserSeederUat extends Seeder
      */
     public function run()
     {
-        $fakeEmailDomain = '@talent.test';
-
         // users with sub values from Sign In Canada, redirecting to uat-talentcloud.tbs-sct.gc.ca
-
         // Ensure we don't create duplicates if users already exist, since we don't truncate tables by default (unlike on local).
         User::updateOrCreate(
-            ['email' => 'tristan-orourke'.$fakeEmailDomain],
+            ['email' => 'usertester923@gmail.com'],
```

Comment:
Really we log in by sub, not email now.  This might be a good chance to switch the lookup around.  It could save some trouble in case someone were to change their email address in the UAT vertical.  Some of these emails are clearly junk, too, and not useful for lookup.

Reply:
Good idea! Fixed by [054fa75](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5313/commits/054fa752aa439cbbee66d5354ce8a5d148e890a7)